src/components/LoginRegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,86 @@
+import { useState, useContext } from "react";
+import { Link } from "react-router-dom";
+import ApiContext from "../../contexts/ApiContext";
+import "./login-register-form.scss";
+
+const RegisterForm = () => {
+  const api = useContext(ApiContext);
+  const [msg, setMsg] = useState(null);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const { target } = event;
+    const [login, password, re_password] = target;
+    if (!login.value || !password.value || !re_password.value) {
+      setMsg({ type: "error", msg: "All inputs required" });
+    } else if (password.value !== re_password.value) {
```

Comment:
do you no need validation for password?

Reply:
will you pull the server with every changed character? and how long will it take you to lay down?checks should be on the client, no need to load the server